firebaseConfig.ts

```diff
@@ -1,6 +1,10 @@
 // Import the functions you need from the SDKs you need
 import { initializeApp } from "firebase/app";
 import { getAuth } from "firebase/auth";
+import { getFirestore, collection, query, orderBy } from "firebase/firestore";
+import firebase from 'firebase/compat/app';
```

Comment:
I think this method of using the compat libraries is being replaced by the modular api so we shld try and use the new import statements instead. I think there's more info here:

https://firebase.google.com/docs/web/modular-upgrade